Event/RouterGenerateEvent.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2014 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Component\Routing\Event;
+
+use Symfony\Component\EventDispatcher\Event;
+
+class RouterGenerateEvent extends Event
+{
+    /**
+     * @var string
+     */
+    protected $name;
```

Comment:
should have been private in the match event too, but there its too late as making it private is a BC break.